opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -237,14 +237,15 @@ def start_span(
         """Starts a span.
 
         Create a new span. Start the span without setting it as the current
-        span in this tracer's context.
+        span in the context. To start the span and use the context in a single
+        method, see :meth:`start_as_current_span`.
 
-        By default the current span will be used as parent, but an explicit
-        parent can also be specified, either a `Span` or a `opentelemetry.trace.SpanContext`. If
-        the specified value is `None`, the created span will be a root span.
+        By default the current span in the context will be used as parent, but an
+        explicit parent can also be specified, either a `Span` or a `opentelemetry.trace.SpanContext`.
```

Comment:
FYI: I think this won't be the case after [this change](https://github.com/open-telemetry/opentelemetry-python/pull/1146) but we can probably update the docs afterwards to reflect this.

Reply:
Good point, I'll make sure to update it in that change.